app/parse.py

```diff
@@ -15,8 +28,49 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def get_products_detail_info(page_url: str, driver: WebDriver) -> list[Product]:
+    driver.get(page_url)
+    cookies = driver.find_elements(By.CLASS_NAME, "acceptCookies")
+
+    if cookies:
+        cookies[0].click()
+
+    more_button = driver.find_elements(By.CLASS_NAME, "ecomerce-items-scroll-more")
+
+    if more_button:
+        while more_button[0].is_displayed():
+            more_button[0].click()
+            time.sleep(0.1)
+
+    soup = BeautifulSoup(driver.page_source, "html.parser")
+    products = soup.select(".thumbnail")
+
+    return [get_single_product(product) for product in products]
+
+
+def get_single_product(product: Tag) -> Product:
+    return Product(
+        title=product.select_one(".title")["title"],
+        description=product.select_one(".description").text,
+        price=float(product.select_one(".price").text.replace("$", "")),
+        rating=len(product.select(".glyphicon-star")),
+        num_of_reviews=int(
+            product.select_one(".ratings > p.review-count").text.split()[0]
+        ),
+    )
+
+
 def get_all_products() -> None:
-    pass
+    with webdriver.Chrome() as driver:
+        for name, url in URLS.items():
+            with open(f"{name}.csv", "w", encoding="utf-8", newline="") as f:
+                writer = csv.writer(f)
+                writer.writerow(PRODUCT_FIELDS)
+                products = get_products_detail_info(url, driver)
+                writer.writerows([astuple(product) for product in products])
```

Comment:
create class, for example `ProductScraper`
and rewrite all this functions to methods